pkg/apis/eksctl.io/v1alpha5/defaults_test.go

```diff
@@ -55,13 +55,12 @@ var _ = Describe("ClusterConfig validation", func() {
 
 	Context("SSH settings", func() {
 
-		It("Providing an SSH key enables SSH", func() {
+		It("Providing an SSH key enables SSH when SSH.Allow not set", func() {
 			testKeyPath := "some/path/to/file.pub"
 
 			testNodeGroup := NodeGroup{
 				VolumeSize: &DefaultNodeVolumeSize,
 				SSH: &NodeGroupSSH{
-					Allow:         Disabled(),
 					PublicKeyPath: &testKeyPath,
 				},
 			}
```

Comment:
can you add in the end of this test please? So we cover for the case I explained above.

```
Expect(*testNodeGroup.SSH.PublicKeyPath).To(BeIdenticalTo(testKeyPath))
```

Reply:
Good idea! :)

Reply:
@treacher Sorry, what I meant was if we can add that `Expect` in this test under line 70 so that we make sure that the ssh key path is not overriden by the default value if we have a bug. 

Can you add that and make sure it's rebased, then I'll approve and merge! :)

Reply:
Ah, I've added it to that test case too :)